test/dialect/oracle/test_types.py

```diff
@@ -323,6 +350,16 @@ def test_interval(self, metadata, connection):
             datetime.timedelta(days=35, seconds=5743),
         )
 
+    def test_interval_literal_processor(self, connection):
```

Comment:
this test does not seem to do what it says. no literal processor is used.

maybe you wanted to use `parameter=literal(..., literal_execute=True)`